### ISSUE TITLE
OnDemand: Add 'Troubleshoot' section

### ISSUE THREAD

**Issue by HenrikBengtsson** [UCSF-CBI/c4#69]
Add a 'Troubleshoot' section to https://www.c4.ucsf.edu/howto/ondemand.html with information on how to access OnDemand session log output, how to report errors (e.g. include log output), ...

**Comment by hgputnam** [comment]
Good idea, I find myself typing the same sentences on Slack quite a few times...
